news/add_fzf_widgets_extension.rst

```diff
@@ -0,0 +1,14 @@
+**Added:** None
```

Comment:
Hey @shahinism -- you need to remove the `None` from this line to resolve the test failures.  Thanks!